contracts/test/IntegrationTestBase.sol

```diff
@@ -853,6 +909,7 @@ contract IntegrationTestBase is Test, TestParams, TestRegistry, TestSubnetActor,
 
     function registerSubnetGW(uint256 collateral, address subnetAddress, GatewayDiamond gw) public {
         GatewayManagerFacet manager = GatewayManagerFacet(address(gw));
+        GatewayGetterFacet getter = GatewayGetterFacet(address(gw));
```

Comment:
I think this is starting to become a bit pedantic. I would like to remove these local variables going forward. How about we create a `GatewayFacetsHelper` library with a bunch of functions that take `GatewayDiamond` or `address` and return the right facet? We can then attach it to the type with `using` and it becomes much nicer.

```
library GatewayFacetsHelper {
    function getter(address gw) returns GatewayGetterFacet;
    function manager(address gw) returns GatewayManagerFacet;
}

// binding
using GatewayFacetsHelper for address;

// usage
gw.getter().getNetworkName()
```

This is a lot easier to read since you always use the name of the entity (gateway) so it's easier to follow what thing this is the getter to.